### PR TITLE
Add module for MaliciousMacroBot

### PR DESCRIPTION
**Pull request by awest1339**
MaliciousMacroBot is a machine learning classifier intended to perform triage on Office documents https://github.com/egaus/MaliciousMacroBot

### CODE REVIEW

**Comment by awest1339** [comment]
@ptcNOP

**Comment by ptcNOP** [comment]
Is [this](https://github.com/mitre/multiscanner/pull/69/files#diff-32d2e47819544fc911262c5ba7230038R44) needed? Doesn't look like you use `p`.

**Comment by awest1339** [comment]
Good catch. Deleted

File: modules/MachineLearning/MaliciousMacroBot.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    results = []
+
+    mmb = MaliciousMacroBot()
+    mmb.mmb_init_model()
+
+    for fname in filelist:
+        # Ensure libmagic returns results
+        if REQUIRES[0] is not None:
+            # only run the analytic if it is an Office document
+            if 'Microsoft' in _get_libmagicresults(REQUIRES[0][0], fname):
```
Comment (emmanvg) [inline modules/MachineLearning/MaliciousMacroBot.py]: Do you know if the tool supports other document file formats like OpenDocument? It seems like they might get excluded based on this check.

Comment (emmanvg) [reply modules/MachineLearning/MaliciousMacroBot.py]: https://wiki.openoffice.org/wiki/Documentation/FAQ/Macros/Can_I_use_my_Microsoft_Office_macros%3F

Comment (awest1339) [reply modules/MachineLearning/MaliciousMacroBot.py]: You are right. From what I can tell, this tool will run on any file but will return a "benign" prediction for anything that is not an office document. However, I didn't test it with any Open Office documents.

**Pull request merged**